### PR TITLE
Fix/Remove Merging orders option (avellaneda strategy) from the code base

### PR DESCRIPTION
**Pull request by mhrvth**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

Removed hanging order merging logic, settings, tests and documentation from the Avellaneda-Stoikov strategy. The functionality was already disabled through default settings.

**Tests performed by the developer**:

Ran all unit tests of the Avellaneda-Stoikov strategy and tests in test_hanging_orders_tracker.py . Ran the hummingbot, created and ran the Avellaneda-Stoikov strategy on Huobi exchange.

**Tips for QA testing**:

There's no new functionality, and no new tests. The code and the tests were only removed.

### CODE REVIEW

File: hummingbot/templates/conf_avellaneda_market_making_strategy_TEMPLATE.yml [hunk 1]
```diff
@@ -44,9 +44,6 @@ add_transaction_costs: null
 # when one side is filled (hanging orders feature) (true/false).
 hanging_orders_enabled: null
 
-## Select way of aggregating hanging orders. Whether if leaving them as they are or calculating a resulting hanging order
```
Comment (aarmoa) [inline hummingbot/templates/conf_avellaneda_market_making_strategy_TEMPLATE.yml:47]: When changing any TEMPLATE.yml it is important to change the "template_version" value (at the beginning of the file)

**Review by aarmoa — CHANGES_REQUESTED** [review]
I added a small comment. Even when the change is small it is required to merge the PR.

**Review by RC-13 — APPROVED** [review]
Tested Avellaneda market-making strategy and didn't encounter any issue. LGTM

**Review by aarmoa — APPROVED** [review]
The changes to remove hanging orders aggregation look good to me

**Pull request merged**